phpBB/includes/functions.php

```diff
@@ -5448,22 +5436,6 @@ function phpbb_user_session_handler()
 }
 
 /**
-* Check if PCRE has UTF-8 support
-* PHP may not be linked with the bundled PCRE lib and instead with an older version
-*
-* @return bool	Returns true if PCRE (the regular expressions library) supports UTF-8 encoding
-*/
-function phpbb_pcre_utf8_support()
```

Comment:
Not sure that's needed. The main reason the functions_compatibility.php script exists is for extension or MOD compatibility. Can't think of a lot of cases this function would be used outside of phpBB in a MOD or extension, unless you know otherwise.